test/recipes/15-test_ecparam.t

```diff
@@ -20,8 +20,22 @@ setup("test_ecparam");
 plan skip_all => "EC isn't supported in this build"
     if disabled("ec") || disabled("ec2m");
 
-my @valid = glob(data_file("valid", "*.pem"));
-my @invalid = glob(data_file("invalid", "*.pem"));
+my @valid = ();
+my @invalid = ();
+
+if (defined $ENV{'fips'}) {
```

Comment:
I think the env var should match the pre-processor, ```FIPS_MODE``` and be set to 1 if on. Sometimes 'unsetenv' / 'unset' is not convenient.

Reply:
The changes here are ahead of the framework ones.  This isn't the way FIPS detection will be done but we don't know how it will be done yet.  Until someone works on the latter, this is good enough.

Reply:
I disagree.  There's no point in making things needlessly different, especially for an interim measure.